### ISSUE TITLE
RGB Images

### ISSUE THREAD

**Issue by Louis-Leee** [rvp-group/vbr-devkit#2]
Hi, thanks for your impressive work!

After I convert the original .bag into kitti format, I have "campus/campus_train0_01_kitti/camera_left/data", where there are only gray images. So I want to know how can I get the RGB images.

Thanks again.

**Comment by EmanueleGiacomini** [comment]
Hi, thanks for reaching out :smile: 
The converter from Bag1 to KITTI currently dumps the images in raw format (Bayer_RGGB8).

I will place here an example in Python to de-bayer the images
```python
import cv2

image_bayer = cv2.imread("camera_left/data/0000000001.png", cv2.IMREAD_UNCHANGED)
image_rgb = cv2.cvtColor(image_bayer, cv2.COLOR_BAYER_RG2RGB)
```

We could provide a flag to automatically de-bayer images during the conversion, but it would dramatically increase the converted dataset's size.

**Comment by Louis-Leee** [comment]
Thanks for your reply and demo code! I will try that.

**Comment by gmberton** [comment]
Hi @EmanueleGiacomini, I think a flag would be highly beneficial, increasing the size should be fine for many users including me

**Comment by EmanueleGiacomini** [comment]
> Hi @EmanueleGiacomini, I think a flag would be highly beneficial, increasing the size should be fine for many users including me

We included your suggestion in commit 987ae4c (published in the new `0.0.9` release). During the conversion in KITTI format, you can use the `--rgb-conversion` flag to enable direct bayer to rgb image transform.

**Comment by gmberton** [comment]
Thank you so much!
